test/suites/config.sh

```diff
@@ -42,6 +121,9 @@ test_config_profiles() {
 
   # test live-adding a nic
   lxc start foo
+  echo "XXX restarting foo"
+  lxc restart foo
```

Comment:
We need "lxc restart foo --force" there otherwise it's going to hang just like stop sometimes does.